helium-lib/src/error.rs

```diff
@@ -13,6 +13,10 @@ pub enum Error {
     Anchor(#[from] anchor_client::ClientError),
     #[error("anchor lang: {0}")]
     AnchorLang(#[from] helium_anchor_gen::anchor_lang::error::Error),
+    #[error("Account already exists")]
+    AccountExists,
+    #[error("Account non existent: {0}")]
```

Comment:
How are we keeping these apart from the account not found erros in the Solana part of this enum?